Bottom: eec4fb7a6d90e6dd80544820d3c1930a628d8201
Top:    3c9ed06ea391248dfbb46165f3b52cb109e1b6b1
Author: Andrew Cooper <andrew.cooper@novatechweb.com>
Date:   2018-07-12 11:12:25 -0500

Fix mariadb bind ip address


---
diff --git a/roles/mariadb/tasks/main.yml b/roles/mariadb/tasks/main.yml
index 70e29ec6..b27a0af7 100644
--- a/roles/mariadb/tasks/main.yml
+++ b/roles/mariadb/tasks/main.yml
@@ -42,15 +42,3 @@
     permanent: true
     state: enabled
   when: mariadb_open_firewall
-- name: Set host ssh ListenAddress
-  template:
-    src: mariadb-bind-address.j2
-    dest: /etc/my.cnf.d/mariadb-bind-address.cnf
-  when: ansible_br0 is defined and ansible_br0.ipv4 is defined and ansible_br0.ipv4.address is defined
-
-- name: Restart MariaDB service
-  service:
-    name: mariadb
-    enabled: yes
-    state: restarted
-  when: ansible_br0 is defined and ansible_br0.ipv4 is defined and ansible_br0.ipv4.address is defined
diff --git a/roles/mariadb/templates/mariadb-bind-address.j2 b/roles/mariadb/templates/mariadb-bind-address.j2
index f30ea73a..831edde0 100644
--- a/roles/mariadb/templates/mariadb-bind-address.j2
+++ b/roles/mariadb/templates/mariadb-bind-address.j2
@@ -1,3 +1,3 @@
 # Bind the database only to the host port
 [mysqld]
-bind-address = {{ ansible_br0["ipv4"]["address"] }}
+bind-address = {{ ansible_default_ipv4["address"] }}
